backend/package.json

```diff
@@ -11,6 +11,7 @@
   "license": "ISC",
   "dependencies": {
     "child_process": "^1.0.2",
+    "cors": "^2.8.5",
     "ejs": "^3.1.9",
```

Comment:
remove ^ from every dependency. we require only a particular version of package

Reply:
done